draft-ietf-rats-architecture.md

```diff
@@ -1304,6 +1306,39 @@ considered.
 The security of conveyed information may be applied at different layers, whether by a conveyance protocol, or an information encoding format. This architecture expects attestation messages (i.e., Evidence, Attestation Results, Endorsements, Reference Values, and Policies) are end-to-end protected based on the role interaction context.
 For example, if an Attester produces Evidence that is relayed through some other entity that doesn't implement the Attester or the intended Verifier roles, then the relaying entity should not expect to have access to the Evidence.
 
+## Handle-based Attestation {#handles-sec}
+
+Handles, described in {{handles}}, can be tampered with, dropped, delayed and
+reordered by an attacker.
+
+An attacker could be either external or belong to the distribution group, for
+example if one of the Attester entities have been compromised.
+
+An attacker who is able to tamper with handles can potentially lock all the
+participants in a certain epoch of choice for ever, effectively freezing time.
+This is problematic since it destroys the ability to ascertain freshness of
+Evidence and Attestation Results.
+
+To mitigate this threat, the transport should be at least integrity protected
+and provide origin authentication.
+
+Dropping handles is equivalent to pinning the victim node to a past epoch, and
+will typically result in a denial of service due to the permanent staleness of
+the Attestation Result or Evidence.
+
+Delaying or reordering handles is equivalent to manipulating the victim's
+timeline at will.  This ability could be used by a malicious actor (e.g., a
+compromised router) to mount a confusion attack where, for example, a Verifier
+is tricked into accepting Evidence coming from a past epoch as fresh, while in
+the meantime the Attester has been compromised.
+
+To mitigate these threats, the transport should provide the ability to detect
+reordering and drop.
+
+However, the delay attack described above can only be thwarted if the target
+has an independent notion of time that can be used to drop handles that are not
+expected.
```

Comment:
Still doesn't completely solve the delay attack since you can't drop handles that you still think are valid.